codeql_ruby/cwe-094/CodeInjection/CodeInjection.rb

```
@@ -0,0 +1,199 @@
+
+    eval(Regexp.escape(code))
+# {/fact}
+
+# {fact rule=autoescape-disabled@v1.0 defects=1}
+
+    # BAD
+
+    ActiveJob::Serializers.deserialize(code)
+  end
+# {/fact}'
+'
```

RuboCop (1.80.2)

[fatal] unterminated string meets end of file
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)